ui/shortcuts/src/main/java/org/cru/godtools/shortcuts/GodToolsShortcutManager.kt

```diff
@@ -178,9 +168,13 @@ class GodToolsShortcutManager @VisibleForTesting internal constructor(
     }
 
     @VisibleForTesting
-    @OptIn(ObsoleteCoroutinesApi::class)
+    @OptIn(ExperimentalCoroutinesApi::class, ObsoleteCoroutinesApi::class)
     internal val updatePendingShortcutsActor = coroutineScope.actor<Unit>(capacity = CONFLATED) {
-        channel.consumeAsFlow().conflate().collectLatest {
+        merge(
+            settings.primaryLanguageFlow,
+            settings.parallelLanguageFlow,
+            channel.consumeAsFlow()
+        ).conflate().collectLatest {
```

Comment:
Or does ```collectLatest``` listen for the changes to the flows? And I assume we merge them here in order for ```collectLatest``` to listen for changes to either flow? And then we run these callbacks accordingly after a change to the flow has occurred.

Reply:
`conflate` and `collectLatest` kind of do the same thing, but at different points during execution.

- `collectLatest` will start processing the most recent value, if a new value shows up before it finishes, it will cancel the previous execution and start processing the new value
- `conflate` will only buffer a single value for execution, if a new value shows up it will drop the old value and buffer the new one